frontend/test/views/AccountView.spec.js

```diff
@@ -75,4 +77,55 @@ describe('AccountView.vue', () => {
       expect(wrapper.vm.$router.push.called).to.be.true;
     });
   });
+
+  it('should toggle the secret value on click', async () => {
+    // Create a stub for the getUser method and return a mock user
+    const mockUser = {
+      clientSecret: 'testsecret',
+    };
+
+    const getUserStub = sandbox.stub(authStore, 'getUser');
+    getUserStub.returns(mockUser);
+
+    const wrapper = getMountedComponent();
+
+    expect(wrapper.vm.showSecretValue).to.be.false;
+    expect(wrapper.vm.secretValue).to.deep.equal(['<click to show>']);
+
+    // Find the section box component for the "Client Secret" section
+    const sectionBox = wrapper.findComponent({ref: 'credentials'});
+
+    // Simulate a click event on the section box
+    await sectionBox.trigger('click');
+
+    // Assert that showSecretValue has been toggled
+    expect(wrapper.vm.showSecretValue).to.be.true;
+
+    // Assert that the computed secretValue property returns the client secret
+    expect(wrapper.vm.secretValue).to.deep.equal([wrapper.vm.user.clientSecret]);
```

Comment:
Instead of checking the value of the computed data, you would need to get the SectionBox component itself and verify that the `content` prop has been updated with the expected value.